cleancat/__init__.py

```diff
@@ -1,4 +1,5 @@
 from cleancat.base import (
+    URL,
```

Comment:
Which linting rule did this sorting? It's not ordered alphabetically and it looks like it was before.

Reply:
Seems like isort (and ruff's version of it treats it as a "constant") https://pycqa.github.io/isort/docs/configuration/options.html#dont-order-by-type